src/main/java/com/mraof/minestuck/blockentity/machine/UraniumPowered.java

```diff
@@ -0,0 +1,13 @@
+package com.mraof.minestuck.blockentity.machine;
+
+/**
+ * Used to keep track of block entities that store uranium fuel and apply external sources of fuel.
+ */
+public interface UraniumPowered
+{
+	void addFuel(short fuelAmount);
+	
+	boolean canBeRefueled();
```

Comment:
Should this function be part of the interface for any reason?

Reply:
I suppose its not necessary, I believe I included it as a way to ensure standardization despite no external uses

Reply:
In the respective machines, it is tied to the machines capability to consume one fuel item. I don't think it'll do us any good to ensure that future uranium powered machines also has this capability. I recommend removing it from the interface.